sql/explain.go

```diff
@@ -48,9 +51,20 @@ func (p *planner) Explain(n *parser.Explain) (planNode, error) {
 	if err != nil {
 		return nil, err
 	}
-	plan, err = markExplain(plan, mode)
-	if err != nil {
-		return nil, fmt.Errorf("%v: %s", err, n)
+	switch mode {
+	case explainDebug:
+		plan, err = markExplain(plan, mode)
```

Comment:
Ok, I did the `markDebug` and `populateExplain` renames, but I resisted making `populateExplain` a method on `valuesNode`. Explaining a plan really seems unrelated to the rest of the `valuesNode` functionality.